lib/lualinux.c

```diff
@@ -569,10 +422,31 @@ static const lunatik_reg_t lualinux_signal[] = {
 	{NULL, 0}
 };
 
+/***
+* Returns the symbolic name of a kernel error number.
+* For example, it converts `2` to `"ENOENT"`.
+*
+* @function errname
+* @tparam integer err The error number (e.g., 2).
+* @treturn[opt] string The symbolic name of the error if found, otherwise `nil`.
+* @usage
+* local name = linux.errname(2)
+* if name then print("Error name:", name) end -- "ENOENT"
+*/
+static int lualinux_errname(lua_State *L)
+{
+	int e = (int)luaL_checkinteger(L, 1);
+	const char *n = lunatik_errname(e);
+	if (n)
+		lua_pushstring(L, n);
```

Comment:
what happens if we just push a NULL here and avoid the code bellow?

Reply:
nop, I meant to check what is pushed on pushstring(L, NULL).. is it an empty string? nil? does it explode? Also, what's the best error treatment here? is it returning nil? should we raise an error? perhaps it's better to return "unknown". Think about the use case case; the scripts that use such API.